sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -77,7 +77,7 @@ export enum PaginationParams {
   SearchCriteria = "searchCriteria",
 }
 
-/** Pagination Options DTO */
+/** Pagination Options */
```

Comment:
While adding or refactoring the comments please ensure that they are in below format.
```ts
/**
 * Pagination Options.
 */
```